test/integration/typescript/test/index.test.js

```diff
@@ -49,6 +49,19 @@ describe('TypeScript Features', () => {
       expect($('#cookies').text()).toBe('{}')
     })
 
+    it('should render the cookies page with cookies', async () => {
```

Comment:
There was no test for `req.cookies` values in getServerSideProps but that is supported today so added a test for it to ensure it doesn't break in the future (and that this change is correct).